brands/shop/clothes.json

```diff
@@ -3299,6 +3299,14 @@
       "shop": "clothes"
     }
   },
+  "shop/clothes|Van Heusen": {
+    "countryCodes": ["us"],
+    "tags": {
+      "brand": "Van Heusen",
+      "name": "Van Heusen",
+      "shop": "clothes"
```

Comment:
This will need a wikidata entry created so it will show up in the editors. You can usually look at one of the entries for a similar business to see useful keys to enter. One of the big ones for us is the social media entries since that powers the logos.

Reply:
I'm not quite sure if I'm understanding wikidata 100%. I looked at a few other brands and it seemed like they had Wikipedia pages and wikidata. Any brand that did not have a Wikipedia page I was not able to find wikidata for. I can take a look at Van Heusen's social media logos to see if there is any kind of wikidata associated with them if that is a source for brands without Wikipedia pages.

Reply:
@tas50, Q67551865 wouldn't work? The link to their website on their Wikidata entry has a section for locating stores. So, I assume its the same thing and that we can just use the normal Wikidata entry.

Reply:
Btw @quetcodesfire, I noticed that a lot of the things your doing PR's for already had ones open by other people. You should really check to make sure there aren't already PR's open for things. Otherwise, we might miss the earlier ones like I did when I merged all yours and that's not so good for the people that technically did them first who don't get theirs merged now. 

Thanks

Reply:
Q67551865 looks ok to me